src/runtime/context.ts

```diff
@@ -1,24 +1,46 @@
-const { Exception } = require('../datatypes/exception');
-const { typeIsArray } = require('../util/util');
-const { NullMessageListener } = require('./messageListeners');
-const dt = require('../datatypes/datatypes');
+import { Exception } from '../datatypes/exception';
+import { typeIsArray } from '../util/util';
+import * as dt from '../datatypes/datatypes';
+import { CodeService } from '../cql-code-service';
+import { MessageListener, NullMessageListener } from './messageListeners';
+import { Patient } from '../cql-patient';
+import { Parameter } from '../types/runtime-types';
+
+export class Context {
+  // Construcor args
+  parent: any;
+  _codeService?: CodeService | null;
+  _parameters?: Parameter;
```

Comment:
IIRC, we used tthe `_` as a convention to mark "private" (or "protected"?) properties (when access modifiers were not available to us).  I just tried adding a `private` access modifier to these locally and everything still seems to work -- so perhaps we should go ahead and do that to make the intention more explicit?